src/fontra/client/core/utils.js

```diff
@@ -221,3 +221,23 @@ export function makeUPlusStringFromCodePoint(codePoint) {
     ? "U+" + codePoint.toString(16).toUpperCase().padStart(4, "0")
     : "";
 }
+
+export function buildShortcut(shorCutRegister) {
```

Comment:
- Please keep this function in context-menu.js for now.
- I would name the function `buildShortCutString()`
- I would name its argument `shortCutDefinition`